src/plugins/vis_augmenter/public/view_events_flyout/actions/view_events_option_action.tsx

```diff
@@ -5,8 +5,7 @@
 
 import { i18n } from '@osd/i18n';
 import { EuiIconType } from '@elastic/eui/src/components/icon/icon';
-import { get } from 'lodash';
-import { CoreStart } from 'opensearch-dashboards/public';
+import { get, isEmpty } from 'lodash';
```

Comment:
nit: There seems to be a push in OSD to not use Lodash anymore. I'm personally not sold on it but just giving a heads up :)